js/downloads.js

```diff
@@ -36,7 +36,8 @@ addRelease("2.4.0", new Date("11/02/2018"), packagesV9, true, true);
 addRelease("2.3.2", new Date("09/24/2018"), packagesV8, true, true);
 addRelease("2.3.1", new Date("06/08/2018"), packagesV8, true, true);
```

Comment:
btw, this should be `packagesV8, true, false);` I think?

Reply:
Yes, but `2.3.1` is still on the mirror.

Reply:
though I tested a few mirror link for 2.3.1, seems like it is still replicated on mirrors..

Reply:
just checked - I could remove it from dist/release later - after this is changed to false.

Reply:
Shall I make another PR to remote this?

Reply:
I think it's safer to put `false` on all "not-the-last" release of the minor release tree, ie. 2.3.1, 2.1.2 etc. it doesn't hurt either way, for now, since 2.3.1 is still on the mirror, but it kinda should be in archive instead

Reply:
That's right, and I was going to update that and remove the older releases from the mirrors just now -- I'll take care of it